server/db.ts

```diff
@@ -0,0 +1,49 @@
+import { Task, User } from '../api'
+
```

Comment:
Why you decided to not use real SQLite?

Reply:
As I understand, there will no any «logux specific» code related to DB, so it will not bring any benefit for this case.

Reply:
It will be hard to play and test Logux without persistence storage.

For instance, to emulate offline, we can stop server. But without SQLite you will lose data by turning off and starting server again, which will lead to data inconsistency and laggy UX.